custom_components/powercalc/data/signify/LCB001/model.json

```diff
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Kasa, KP115(US)",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v0.24.8:docker"
+    },
+    "name": "\tHue White and Color Ambiance BR30 Bluetooth",
```

Comment:
```suggestion
    "name": "Hue White and Color Ambiance BR30 Bluetooth",
```

Reply:
Suggestion won't apply even, hmm. look like the actual code in the branch is ok.

Reply:
Looked at the network tab in browser and even a 500 internal server error is thrown on the `apply_suggestion` call. So I assume it will be allright when the MR is merged. If not, we will fix it later.